scripts/generate_doc.sh

```
@@ -8,40 +40,43 @@ set -e
-echo '::group::{generate_openapi_html}'
+start_group "Generate openapi html with redocly"
 docker run --rm \
-    -v $(pwd)/data:/data -v $(pwd)/gh_pages/:/output \
-    ghcr.io/redocly/redoc/cli:latest \
-    build -o /output/users/ref-openapi/index.html searchalicious-openapi.yml
-sudo chown $UID -R gh_pages/users/ref-openapi
-echo "::endgroup::"
-
-echo "Generate webcomponents documentation"
-echo '::group::{generate_custom_elements}'
+  -v $(pwd)/data:/data \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)